tests/unit/services/sl-behavior-test.js

```diff
@@ -218,114 +152,65 @@ test( 'isAble() requires at least two parameters to be provided', function( asse
 });
 
 test( 'isAble() requires first parameter to be a String', function( assert ) {
-    // Empty parameter
-
-    let assertionThrown = false;
-
-    try {
-        BS.isAble();
-    } catch( error ) {
-        assertionThrown = true;
-    }
 
-    assert.ok(
-        assertionThrown,
-        'Parameter was empty'
-    );
-
-    // null parameter
-
-    assertionThrown = false;
-
-    try {
-        BS.isAble( null, 'notUnderTest' );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was null'
-    );
-
-    // Number parameter
-
-    assertionThrown = false;
-
-    try {
-        BS.isAble( 4, 'notUnderTest' );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    // Array Parameter
+    let callIsAble = () => BS.isAble( [], 'notUnderTest' );
```

Comment:
Yes the second param is required. I already have it in the string test, but it looks like it should be in the undefined test so I'm adding it there.